idealalphaPackage.R

```diff
@@ -0,0 +1,201 @@
+library(pwr)
+library(ggplot2)
+
+#functions to justify alpha levels.
+#first run all the code. In the beginning are functions incoroporated in later functions
+
+#call idealalpha(n, effsize, prior, test = "t-test", type = "two.sample", alternative = "two.sided", k = 4)
+#to calculate the alpha level which gives optimal learning
+#the output will be alpha, power and info, which is the expected learning
+
+#call alphaConfirmation(n, effsize, prior, postdis, test = "t-test", type = "two.sample", alternative = "two.sided", k = 4)
+#to calculate the alpha level needed to achieve a given posterior after a significant result
+#the output will be alpha, power, info, and confirm, which is the probability after a positive result
+
+#alphaDisconfirmation(n, effsize, prior, postdis, test = "t-test", type = "two.sample", alternative = "two.sided", k = 4)
+#to calculate the alpha level needed to achieve a given posterior after a non-significant result
+#the output will be alpha power and disconfirm, which is the probability after a negative result
+
+#probpower(effsize, prior, postdis, postconf, test = "t-test", type = "two.sample", alternative = "two.sided", k = 4)
+#conduct a probability based power analysis. Here posterior after significance and posterior after non-signficance are fixed
+#the function will then return the sample size and the alpha level needed
+#the output will be alpha, power, disconfirm, confirm, info and n, the sample size per group
+
+
+
+#The next functions are defined to be part of the four main functions
+
+#calculate the probability that the hypothesis is true, after confirming
+#' @export
+postconfirm <- function(power,alpha,prior)
+                          {
+        (power*prior)/(power*prior+alpha*(1-prior))
+}
+
```

Comment:
Hey Maxi, 
I think your code looks great already! 
One thing I noticed is that you could improve your coding style a little bit. 

- Leave spances between infix operators (=, +, -, <-, etc.) 
- Place a space before left parentheses (except in a function call)
- don't leave curly brackets on it's own